src/app/core/models/content-configuration-parameter/content-configuration-parameter.interface.ts

```diff
@@ -1,4 +1,5 @@
 export interface ContentConfigurationParameterData {
   definitionQualifiedName: string;
   value: string | object;
+  type?: string;
```

Comment:
`type` property does not need to be nullable. ICM server guarantees the presence of that property.
```
        cpRO.setType(configParam.getConfigurationParameterType()
                        .getQualifiedName());
```

Reply:
Hi Johannes.
I was a bit conservative because I do not want to change the model interface.
I patch this on the next commit